wiki/Tournaments/CMT_4K/2018/en.md

```diff
@@ -55,34 +55,83 @@ The Chinese osu!mania 4K Tournament 2018 is run by various community members by
   - bilibili: [aiyulu](https://live.bilibili.com/10183) (Primary), [Flow Light](https://live.bilibili.com/73668) (Secondary), [Rasis](https://live.bilibili.com/5333) (Backup).   
   - Twitch: [aiyulu](https://www.twitch.tv/aiyulu).
 - [Beatmap suggestion](https://docs.google.com/forms/d/e/1FAIpQLSeGjHNxIOw1cOfj1NHn1KwbXBfa5mZnqRNQQnFT-v6z6uRJYg/viewform)
+- [Tournament Statistics Sheet](https://docs.qq.com/sheet/DSHZVSUJ3TGFRY3Fj)
```

Comment:
Tournament Statistics Sheet -> Statistics Sheet
Make it Bold and do it for the ones that on the Match Results